test/PrintAsObjC/classes.swift

```diff
@@ -318,6 +318,13 @@ typealias AliasForNSRect = NSRect
   func testBridgingOptionality(_ a: UnsafePointer<Int>?, b: UnsafeMutablePointer<Int>!, c: AutoreleasingUnsafeMutablePointer<Methods?>?) {}
 }
 
+// CHECK-LABEL: IB_DESIGNABLE
+// CHECK-LABEL: @interface MyDesignableObject : NSObject
```

Comment:
Ah, the case I'm concerned about looks like this:

```
IB_DESIGNABLE // first CHECK-LABEL matches here
SWIFT_RUNTIME_NAME("...")
@interface SomeUnrelatedObject
@end

SWIFT_RUNTIME_NAME("...")
@interface MyDesignableObject : NSObject // next CHECK-LABEL matches here
```

The only way to tell that the two have anything to do with each other is to use CHECK-NEXT so that we know what's exactly in between. I think this would be sufficient:

```
// CHECK-LABEL: IB_DESIGNABLE
// CHECK-NEXT: SWIFT_RUNTIME_NAME(
// CHECK-NEXT: @interface MyDesignableObject : NSObject
```